sdstore/sdstore.go

```diff
@@ -32,6 +33,24 @@ type sdStore struct {
 	client *retryablehttp.Client
 }
 
+// getExpectContinueTimeout checks the SD_EXPECT_CONTINUE_TIMEOUT environment variable.
+// It returns the timeout. If the variable is not set, invalid, or negative, it defaults to 1 sec.
+func getExpectContinueTimeout() int {
+	envValue := os.Getenv("SD_EXPECT_CONTINUE_TIMEOUT")
+
+	if envValue == "" {
+		// default 1 sec
+		return 1
+	}
+
+	expectcontinuetimeout, err := strconv.Atoi(envValue)
```

Comment:
Can we use camelcase here too
```suggestion
	expectContinueTimeout, err := strconv.Atoi(envValue)
```

Reply:
Thank you for your review. I have fixed it.